src/containers/sidebars/data-global-sidebar/analyze-areas-sidebar-card/index.js

```diff
@@ -88,10 +88,18 @@ const AnalyzeAreasContainer = (props) => {
   }
 
   const onShapeUploadError = (error) => {
-    setPromptModalContent({
-      title: WARNING_MESSAGES[error.details.httpStatus].title,
-      description: WARNING_MESSAGES[error.details.httpStatus].description(),
-    });
+    if (error.message === "Invalid file format.") {
+      setPromptModalContent({
+        title: WARNING_MESSAGES.file.title,
+        description: WARNING_MESSAGES.file.description(),
+      });
+    }
```

Comment:
Maybe we can use an else statement here. I'm not very happy about using the error message. But I don't think we've got something else